apps/apollo-stories/src/components/Link/render.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "@axa-fr/design-system-apollo-react";
+import { type Args } from "storybook/internal/types";
+
+const LinkBlock = () => (
+  <div>
+    <Link href="https://example.com" />
+  </div>
+);
+
+export const renderLinkAll = () => (
+  <div className="af-link-demo">
+    <LinkBlock />
+  </div>
+);
+
+export const renderLink = ({
+  href,
+  openInNewTab,
+  leftIcon,
+  rightIcon,
+  className,
+  classModifier,
+  ...args
+}: Args) => (
+  <Link
+    openInNewTab={openInNewTab}
+    leftIcon={leftIcon}
+    rightIcon={rightIcon}
+    className={className}
+    classModifier={classModifier}
+    {...args}
+    href={href}
+  />
+);

```

Comment:
Du coup il y a vraiment besoin d'une fonction renderLink ?
```suggestion
export const renderLink = (args: Args) => <Link {...args} />;
```